### PR TITLE
2022-05-20 release

### PR DESCRIPTION
**Pull request by shawntanzk**
Updated imports too

### CODE REVIEW

**Comment by shawntanzk** [comment]
I'm guessing `property_value: http://purl.org/dc/terms/contributor -> relationship: dc-contributor` is correct

**Comment by shawntanzk** [comment]
Tagging @PaulNSchofield

File: pato-base.obo [hunk 1]
```diff
@@ -11929,35 +11933,43 @@ relationship: is_opposite_of PATO:0001742 ! increased radioactivity
...
+synonym: "radiation exposed" EXACT []
+is_a: PATO:0002198 ! quality of a substance
 
 [Term]
 id: PATO:0001745
-name: radiation absorbed dose
-def: "A radiation exposure quality inhering in a substance by virtue of the radiation energy \"deposited\" in a kilogram of a substance." [arpansa:arpansa]
+name: obsolete radiation absorbed dose
+def: "OBSOLETE A radiation exposure quality inhering in a substance by virtue of the radiation energy \"deposited\" in a kilogram of a substance." [arpansa:arpansa]
+comment: see: https://github.com/pato-ontology/pato/issues/485
```
Comment (matentzn) [inline pato-base.obo:11948]: dont know who did that but please just use term tracker item, not this weird construct, in the future

**Review by matentzn — APPROVED** [review]
Cant see any issues.

File: pato-base.obo (hunk 1 above)
Comment (shawntanzk) [reply pato-base.obo:11948]: whoops that's me, sorry about that >.< will fix it for next release

**Comment by reality** [comment]
We have the changes we need to make to RBO queued up for this, which we'll release immediately after. So, looking forward to the PATO release :)) Thanks a lot!

**Comment by shawntanzk** [comment]
@reality - just pushed release through, should all work on your end now. Thanks

**Pull request merged**